examples/wordpress/pod-wordpress.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: wordpress
+  labels: 
+    name: wordpress
+spec: 
+  containers: 
+    - image: wordpress
+      name: wordpress
+      env:
+        - name: WORDPRESS_DB_USER
+          value: wp_user
+        - name: WORDPRESS_DB_PASSWORD
+          value: wp_pass
+        - name: WORDPRESS_DB_NAME
+          value: wp_db
+        - name: WORDPRESS_DB_HOST
+          # this is the name of the mysql service fronting the mysql pod in the same namespace
+          # expands to mysql.default.svc.cluster.local  - where 'default' is the current namespace
```

Comment:
`expands to mysql.<namespace>.svc.cluster.local - where 'default' is the current namespace`